### DISCUSSION TITLE
Add custom classes for Past & Future dates

### DISCUSSION THREAD

**Discussion by azhr-m** [gpbl/react-day-picker#2000] [Support] [answered]
I can currently apply custom classes for `today`, `outside` and `booked` dates; however, I need to extend this functionality to include custom classes for `past` and `upcoming` dates.
```
modifiersClassNames={({
	booked: 'highlighted',
	today: 'today-custom-class',
	outside: 'outside-month-date',
	"past": 'past',
	"upcoming": 'upcoming"
})}
```

**Comment by gpbl** [comment, marked as answer]
@Azhr-Mughl you need to add "modifiers" for past/future 

```tsx
<DayPicker
  modifiers={{
    past: { before: new Date() },
    future: { after: new Date() },
    // upcoming: { after: new Date() }, // as per your code
  }}
  modifiersClassNames={{
     past: "past-date",
     future: "future-date",
     // upcoming: "upcoming", // as per your code
  }}
/>
```

See this example: 

https://codesandbox.io/p/sandbox/react-day-picker-forked-nftt8s?file=%2Fsrc%2FApp.tsx

where past days are blue and future days are red:

<img width="346" alt="Screenshot 2023-12-23 at 6 19 57 AM" src="https://github.com/gpbl/react-day-picker/assets/120693/abc9fc56-4504-43fd-8c0c-4387832fcb6c">
